### ISSUE TITLE
[bug]: removing towers is disallowed while a new session negotiation is in progress

### ISSUE THREAD

**Issue by aphex3k** [lightningnetwork/lnd#7248]
### Background

I know some of the towers I configured are permanently offline and I want to remove them from my towers list. `lncli wtclient towers` correctly shows all configured towers (4). None of them is having an active session as all show `"active_session_candidate": false`. Attempting to remove ANY of the configured towers always yields the same result:
```
$ lncli wtclient remove <redacted>
[lncli] rpc error: code = Unknown desc = removing towers is disallowed while a new session negotiation is in progress
```

```
for i in <redacted> <redacted> <redacted> <redacted>
for> do
for> lncli wtclient remove $i
for> done
[lncli] rpc error: code = Unknown desc = removing towers is disallowed while a new session negotiation is in progress
[lncli] rpc error: code = Unknown desc = removing towers is disallowed while a new session negotiation is in progress
[lncli] rpc error: code = Unknown desc = removing towers is disallowed while a new session negotiation is in progress
[lncli] rpc error: code = Unknown desc = removing towers is disallowed while a new session negotiation is in progress

```
This has been observed for the last several month on different versions of LND. I found #5361 and #7025 but I couldn't figure out how to resolve this issue from there. I tried several times throughout different times of days before and after lnd upgrades to no avail.

### Your environment

* version of `lnd`: 0.15.5-beta commit=v0.15.5-beta
* which operating system (`uname -a` on *Nix): debian/docker (lightninglabs/lnd:v0.15.5-beta)
* version of `btcd`, `bitcoind`, or other backend: bitcoind

### Expected behaviour

I expect the removal of (dead) towers eventually.

### Actual behaviour

The session negotiation seems to never stop/complete.

```
$ lncli wtclient towers
{
    "towers": [
        {
            "pubkey": "<redacted>",
            "addresses": [
                "<redacted>.onion:9911"
            ],
            "active_session_candidate": false,
            "num_sessions": 2,
            "sessions": [
            ]
        },
        {
            "pubkey": "<redacted>",
            "addresses": [
                "<redacted>.onion:9911"
            ],
            "active_session_candidate": false,
            "num_sessions": 0,
            "sessions": [
            ]
        },
        {
            "pubkey": "<redacted>",
            "addresses": [
                "<redacted>:9911",
                "<redacted>.onion:9911"
            ],
            "active_session_candidate": false,
            "num_sessions": 0,
            "sessions": [
            ]
        },
        {
            "pubkey": "<redacted>",
            "addresses": [
                "<redacted>.onion:9911",
                "<redacted>.onion:9911"
            ],
            "active_session_candidate": false,
            "num_sessions": 0,
            "sessions": [
            ]
        }
    ]
```

**Comment by Roasbeef** [comment]
This was fixed in: https://github.com/lightningnetwork/lnd/pull/7025

This is only in master, and will be released with 0.16.

**Comment by aphex3k** [comment]
Nevermind then I will wait till 16 then, sorry for not catching up on that...
